### ISSUE TITLE
csv2 should walk new users through setup stage

### ISSUE THREAD

**Issue by rseuster** [hep-gc/cloudscheduler#121]
for a new user, csv2 should provide more hints what needs to be setup. E.g.:
```
seuster@heplw62:~/Git/cloudscheduler$ rm -rf ~/.csv2/
seuster@heplw62:~/Git/cloudscheduler$ ./cli/bin/cloudscheduler cloud list
Error: user settings for server "prod" does not contain a URL value.
```
should already provide hints one needs -sa, -su and -spw, similar to:
```
seuster@heplw62:~/Git/cloudscheduler$ ./cli/bin/cloudscheduler cloud list -sa https://csv2-dev.heprc.uvic.ca
***
*** Please identify the URL (-sa | --server-address) of the server with which you wish to communicate. Servers
*** require either certificate (-sC | --server-grid-cert, -sK | --server-grid-key) or username/password (-su |
*** --server-user, -spw | --server-password) authentication. These options can be saved for multiple servers
*** by name (there is always a "default" server) using the following command:
***
***     cloudscheduler defaults set -s <sever_name> -sa <server_address> ...
***
*** Subsequently, commands will be directed to the last server selected via the (-s | --server)
*** argument.
***
```

**Comment by crlb** [comment]
fixed dd777dab9de0ede412eca5e5f21caa001367e7d9

**Comment by rseuster** [comment]
closing
